tests/lib/eslint/eslint.js

```diff
@@ -6954,6 +6954,13 @@ describe("ESLint", () => {
                         parser: {
                             parse(text, parserOptions) {
                                 resolvedParserOptions = parserOptions;
+                                return {
+                                    comments: [],
+                                    tokens: [],
+                                    type: "Root",
+                                    loc: {},
+                                    range: []
+                                };
```

Comment:
```suggestion
                                return espree.parse(text, parserOptions);
```

We could use `espree` to return a valid AST here as well (just need to add `const espree = require("espree");` too).